packages/components/src/components/unhandled-error-modal/unhandled-error-modal.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Icon from '../icon';
+import DesktopWrapper from '../desktop-wrapper';
+import MobileWrapper from '../mobile-wrapper';
+import Modal from '../modal';
+import Text from '../text';
+import Button from '../button';
+import MobileDialog from '../mobile-dialog';
+import { localize, Localize } from '@deriv/translations';
+
+const ModalContent = () => (
+    <div className='unhandled-error'>
+        <Icon icon='IcAccountError' size={96} />
+        <Text
+            className='da-icon-with-message__text'
+            as='p'
+            size={'s'}
+            color='general'
+            line_height='xxl'
+            align='center'
+            weight='bold'
+        >
+            {localize('Sorry for the interruption')}
```

Comment:
<img width="359" alt="image" src="https://user-images.githubusercontent.com/56330681/228723051-b0dabf87-40ca-4e2b-954d-ca4141baac3e.png">
localize in the components package won't be extracted out 😅

Reply:
Didn't knew this. Thanks Bala